client/app/hosting/general-informations/GENERAL_INFORMATIONS.controller.js

```diff
@@ -1,25 +1,61 @@
 angular
     .module("App")
     .controller("GeneralInformationsCtrl", class GeneralInformationsCtrl {
-        constructor ($scope) {
+        constructor ($scope, translator) {
             this.$scope = $scope;
+            this.translator = translator;
         }
 
         canOrderSSL () {
-            return !this.$scope.hosting.hasHostedSsl && this.$scope.ssl == null;
+            const canOrderSSL = !this.hasHostedSSL() && !this.hasNonHostedSSL();
```

Comment:
IMO avoid using const is better in this context

Reply:
I do it for test purposes : it helps me add a space where I can set a breakpoint. If I don't have this variable, I can't know what the return value will be, except if I use the console, which is less than ideal !

Thanks for your input though, it is valuable :smiley:

Reply:
@whispyy as nobody likes my variables, I'll get rid of them :smile: